galaxy_ng/app/constants.py

```diff
@@ -48,12 +54,24 @@ class DeploymentMode(enum.Enum):
         "global_description": _("Upload collections to any existing namespace."),
         "ui_category": _("Collection Namespaces"),
     },
+    "ansible.view_collection": {
+        "name": _("View collection"),
+        "object_description": _("View this collection."),
+        "global_description": _("View any existing collection."),
+        "ui_category": _("Collections"),
+    },
     "ansible.delete_collection": {
         "name": _("Delete collection"),
         "object_description": _("Delete this collection."),
         "global_description": _("Delete any existing collection."),
         "ui_category": _("Collections"),
     },
+    # "ansible.view_ansiblerepository": {
```

Comment:
Why did you comment this one out? We do support private repositories, so view permissions here should matter.

Reply:
Because this data structure is a dictionary and the key can only have one instance, which makes it impossible to add the same key to multiple categories.

Reply:
Why does this need to be added to multiple categories?